src/NLog/Common/InternalLogger.cs

```diff
@@ -284,6 +291,36 @@ private static void Write([CanBeNull]Exception ex, LogLevel level, string messag
             }
         }
 
+        private static bool IsSeriousException(Exception ex)
+        {
+            return ex != null && ex.MustBeRethrownImmediately();
+        }
+
+        private static bool NoLogger()
+        {
+            return string.IsNullOrEmpty(LogFile) && !LogToConsole && !LogToConsoleError && LogWriter == null && !WriteToDiagnostics;
+        }
+
+        private static void LogToDiagnostics(LogLevel level, string msg)
+        {
+#if !SILVERLIGHT && !__IOS__ && !__ANDROID__
+            // log to System.Diagnostics.Debug / System.Diagnostics.Trace
```

Comment:
Please add the line above the comment box to the xml docs of the method (`<summary>`) 

Every method shoukd have a `<summary>`